lib/foreman_remote_execution/engine.rb

```diff
@@ -103,11 +103,14 @@ class Engine < ::Rails::Engine
       (Taxonomy.descendants + [Taxonomy]).each { |klass| klass.send(:include, ForemanRemoteExecution::TaxonomyExtensions) }
 
       User.send(:include, ForemanRemoteExecution::UserExtensions)
-      (Host::Base.descendants + [Host::Base]).each do |klass|
+
+      Host::Base.descendants.each do |klass|
```

Comment:
My assumption is the `descendants` is empty by the time of running this. Why not including `[Host::Base]` in the iteration? IIRC the reason why it was originally was to handle both cases, when the subclasses were already loaded, as well as when they are still waiting to be loaded.